.github/test-instructions.md

```diff
@@ -0,0 +1,155 @@
+# Test Guidelines
+
+When writing tests, always run the tests to make sure they pass. Additionally, always run the tests without the watch mode so it doesn't hang up the terminal.
+
+## Setup
+```typescript
+import { describe, it, expect, vi, beforeEach, afterEach } from 'vitest';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+```
+
+## File Structure
+- Place tests in `__tests__` directory next to component
+- Name test files as `ComponentName.test.tsx`
+- Group related tests with describe blocks
+
+## Component Testing
+
+### Basic Tests
+```typescript
+it('renders with default props', () => {
+  render(<Button>Click me</Button>);
+  expect(screen.getByText('Click me')).toBeInTheDocument();
+});
+
+it('renders with variants and sizes', () => {
+  render(<Button variant="primary" size="lg">Large Button</Button>);
+  const button = screen.getByText('Large Button');
+  expect(button).toHaveClass(styles.primaryButton);
+  expect(button).toHaveClass(styles.lgButton);
+});
```

Comment:
Are we changing to use `it` instead of `test` for vitest now? :thinking:

Reply:
Nope, not important. I just want to know what our direction is so I can start adopting it for other project. Since I'm still learning with the agent, but my level was not fully there with the agent yet 😅